src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -58,9 +58,9 @@
     @NotNull
     private static final ThreadLocalisedExceptionHandler DEBUG = new ThreadLocalisedExceptionHandler(Slf4jExceptionHandler.DEBUG);
 
-    private static final long MAX_DIRECT_MEMORY = maxDirectMemory0();
     private static final int JVM_JAVA_MAJOR_VERSION = getMajorVersion0();
     private static final boolean IS_JAVA_9_PLUS = JVM_JAVA_MAJOR_VERSION > 8;
+    private static final long MAX_DIRECT_MEMORY = maxDirectMemory0();
```

Comment:
This re-ordering is necessary as otherwise `maxDirectMemory0` uses incorrect `IS_JAVA_9_PLUS` value. This is really dangerous as some IDE changes order of fields automatically. So this is very error prone.